srcpkgs/arma3-unix-launcher-git/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'arma3-unix-launcher-git'
+pkgname=arma3-unix-launcher-git
+version=20200112
```

Comment:
just use the versions [they are](https://github.com/muttleyxd/arma3-unix-launcher/releases). version=235